install/upgrade_to_2.13.php

```diff
@@ -248,4 +249,16 @@ protected function migrateFkToUnsignedInt() {
       }
       $this->migration->changeField($table, 'id', 'id', 'int ' . DBConnection::getDefaultPrimaryKeySignOption() . ' not null auto_increment');
    }
+
+   protected function addFormVisibilityRestriction(): void {
+      $tables = [
+         'glpi_plugin_formcreator_forms_users' => User::class,
+         'glpi_plugin_formcreator_forms_groups' => Group::class,
+      ];
+
+      foreach ($tables as $table => $class) {
+         $this->migration->createLinkTable($table, PluginFormcreatorForm::class, $class);
+         $this->migration->addField($table, 'uuid', 'string');
+      }
+   }
```

Comment:
The upgrade executes the SQL content in  plugin_formcreator_empty.sql . If the tables don't exist yet, they should be created. This method is useless.

Reply:
Good to know, removed in 290341f5.